### DISCUSSION TITLE
GPT-3 terminal

### DISCUSSION THREAD

**Discussion by TypicalUser1991** [jucasoliveira/terminalGPT#33] [General]
Hey, 
I love the idea of having an ai in your terminal all ready to go, it's as if you have a companion.
But I have had questions. **Where exactly does this gpt ai get its information from?** Because it seems like it's not directly from openai.
Give it simple orders or commands it will start calling you a bitch. Additionally, I thought it would also be cool for this ai to learn more about you and have human-like traits, but thats just an idea. For now my concern is wondering where the ai is pulling it's data from.

**Comment by jucasoliveira** [comment]
@TypicalUser1991 terminalGPT uses openAI own [nodejs sdk](https://beta.openai.com/docs/api-reference/introduction) package to make requests from openAI API's.

The package uses [competition](https://beta.openai.com/docs/api-reference/completions/create) API to request to generate the chat and, we will use [fine-tunning](https://beta.openai.com/docs/api-reference/fine-tunes/create) to improve the conversation.

This project doesn't work if you don't have OpenAI API token.

If you have more questions, please let me know :)
